manager/integration/tests/common.py

```diff
@@ -4680,3 +4681,103 @@ def get_volume_running_replica_cnt(client, volume_name):  # NOQA
             client, volume_name, node.name, chk_running=True)
 
     return cnt
+
+
+def wait_for_orphan_count(client, number, retry_counts=120):
+    for _ in range(retry_counts):
+        orphans = client.list_orphan()
+        if len(orphans) == number:
+            break
+        time.sleep(RETRY_BACKUP_INTERVAL)
+    return len(orphans)
+
+
+def wait_for_file_count(path, number, retry_counts=120):
+    for _ in range(retry_counts):
+        count = exec_nsenter("ls {} | wc -l".format(path))
+        if int(count) == number:
+            break
+        time.sleep(RETRY_INTERVAL_LONG)
+
+    count = exec_nsenter("ls {} | wc -l".format(path))
+    return int(count)
+
+
+def generate_random_id(num_bytes):
+    return ''.join(random.choice(string.ascii_lowercase + string.digits)
+                   for _ in range(num_bytes))
+
+
+def wait_for_orphan_delete(client, name):
+    for _ in range(RETRY_COUNTS):
+        orphans = client.list_orphan()
+        found = False
+        for orphan in orphans:
+            if orphan.name == name:
+                found = True
+                break
+        if not found:
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert not found
+
+
+def delete_orphan(client, orphan_name):
```

Comment:
This is probably better to stay in test_orphan.py first until it's actually reusable in other modules, then we can move to common.py. The same idea applies to other new added functions. 

In general, some refactoring needs for common.py :)